lib/commands/context.js

```diff
@@ -87,6 +87,13 @@ extensions.getNewRemoteDebugger = async function getNewRemoteDebugger () {
   }, this.isRealDevice());
 };
 
+/**
+ * Set context
+ *
+ * @param {?string} name - The name of context to set. It could be 'null' as NATIVE_WIN.
```

Comment:
is https://github.com/appium/appium-xcuitest-driver/pull/1111/files#diff-547d0aab72a72a730ccf5b77092ea1caR127 safe?

Reply:
Yes, since `isNativeContext(name))` will be `true` if the `name` is `null`.
(But let me update there in the case we remove this isNativeContext accidentally)